docs/graasp-insights-user-guide.md

```diff
@@ -0,0 +1,243 @@
+# Datasets
+
+## Menu navigation
+
+When opening Graasp-Insights, you are greeted with the `Datasets` tab.
+Use the top-left menu button to navigate to each distinctive tab.
+
+![Datasets tab with menu open](./figures/datasets-menu.png)
+
+## Importing a dataset
+
+Load a dataset into Graasp Insights by using the lower right `+` button.
```

Comment:
Load a dataset into Graasp Insights _by using the  `+` button on the lower right._